packages/xrpl/HISTORY.md

```diff
@@ -12,6 +12,7 @@ Subscribe to [the **xrpl-announce** mailing list](https://groups.google.com/g/xr
   * `Client.prepareTransaction`
   * `getSignedTx`
   * `isAccountDelete`
+* BREAKING CHANGE: `dropsToXRP` and `Client.getXrpBalance` now return a `number` instead of a `string`
```

Comment:
It's already in the `Breaking Changes` section
```suggestion
* `dropsToXRP` and `Client.getXrpBalance` now return a `number` instead of a `string`
```

Reply:
good catch.  I copied it from the commit subject.